### PR TITLE
BSRD-286: Set the addons to uninstallable

### PR DESCRIPTION
**Pull request by Tonow-c2c**
Related to https://jira.camptocamp.com/browse/BSRD-286

### CODE REVIEW

**Comment by guewen** [comment]
You can remove the "test_base_fileurl_field" test in travis.yml

File: .travis.yml [hunk 1]
```diff
@@ -19,8 +19,6 @@ env:
   - LINT_CHECK="1"
   - TESTS="1" ODOO_REPO="odoo/odoo" EXCLUDE=test_base_fileurl_field
```
Comment (grindtildeath) [inline .travis.yml]:
```suggestion
   - TESTS="1" ODOO_REPO="odoo/odoo"
```

File: .travis.yml [hunk 2]
```diff
@@ -19,8 +19,6 @@ env:
...
   - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE=test_base_fileurl_field
```
Comment (grindtildeath) [inline .travis.yml]:
```suggestion
   - TESTS="1" ODOO_REPO="OCA/OCB"
```

**Review by grindtildeath — APPROVED** [review]
I guess this was @guewen 's request :wink:

**Review by guewen — APPROVED** [review]

**Pull request merged**